corehq/apps/reminders/tests/test_recipient.py

```diff
@@ -65,9 +65,9 @@ def test_recipient_case_owner_location_parent(self):
                 self.assertEqual(reminder.recipient, [parent_location])
 
             # Remove parent location
-            parent_location.delete()
             child_location.parent = None
             child_location.save()
+            parent_location.delete()
```

Comment:
ok, if .delete() is a cascading delete then i think we can just remove this line altogether, remove the child_location.delete() line below, and add a line for self.addCleanup(parent_location.delete) right after creating parent_location